From: SeongJae Park <sj@kernel.org>
Date: Tue, 22 Oct 2024 11:50:14 -0700
Subject: [PATCH] ==== fixup DAMON {self,kunit} tests ====

damon/{self,kunit}tests: minor cleanups and fixups of selftests and kunit tests

Fixup small broken window panes in DAMON selftests and kunit tests.

First four patches make outputs of DAMON debugfs interface selftests
cleaner, by fixing segmentation fault of a test program (patch 1),
removing unnecessary debugging messages (patch 2), and hiding error
messages from expected failures (patches 3 and 4).

Following two patches fixes copy-paste mistakes made documentation
errors on DAMON kunit tests Kconfig (patch 5) and source code (patch 6).

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/wUGOR8M0 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/wUGOR8M0

diff --git a/damon_meta_changes/wUGOR8M0 b/damon_meta_changes/wUGOR8M0
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

